src/components/BotSource/BotSource.tsx

```diff
@@ -0,0 +1,302 @@
+                )
+              })}
+            </TabList>
+            <div className="flex-1 p-6">
+              <TabContent value="Links" key="Links" id="1">
+                <SourceLink />
+              </TabContent>
+              <TabContent value="Sitemaps" key="Sitemaps" id="2">
+                <SourceSitemap />
+              </TabContent>
+            </div>
+          </Tabs>
+        </div>
+      </Card>
+      <div className="p-4 min-w-[48rem]">
+        <div>
+          {Object.values(DATA_TABS).map((tab) => {
+            return (
+              <Button
+                className={`focus:ring-gray-200$ rounded-md  px-4 py-2 text-sm text-gray-700 hover:bg-gray-100 focus:outline-none focus:ring-2 ${dataType === tab.title ? 'bg-blue-400' : 'bg-white'}`}
+                key={tab.title}
+                onClick={() => {
+                  handleChangeDataType(tab.title)
+                }}
+                color="black"
+              >
+                {tab.title}
+              </Button>
+            )
+          })}
+        </div>
+        <Table
+          columns={[
+            {
+              accessorKey: 'title',
+              cell: (data) => {
+                return (
+                  <div>
+                    <span className="text-sm text-black font-semibold text-wrap">
+                      {data.row?.original.title}
+                    </span>
+                  </div>
+                )
+              },
+              header: 'Title',
+            },
+            {
+              accessorKey: 'url',
+              cell: (data) => {
+                return (
+                  <div>
+                    <span className="text-sm text-black font-semibold text-wrap max-w-9 whitespace-nowrap text-ellipsis overflow-hidden">
+                      {data.row?.original.url}
+                    </span>
+                  </div>
+                )
+              },
+              header: 'Url',
+            },
+            {
+              accessorKey: 'typeId',
+              header: 'File type',
+              cell: (data) => {
+                return (
+                  <span
+                    className={`text-sm text-white font-semibold px-2 py-1 rounded-md ${
+                      data.row?.original.typeId === 1
+                        ? 'bg-blue-500'
+                        : 'bg-yellow-500'
+                    }`}
+                  >
+                    {data.row?.original.typeId === 1 ? 'Link' : 'File'}
+                  </span>
+                )
+              },
+            },
+            {
+              accessorKey: 'statusId',
+              header: 'Status',
+              cell: (data) => {
+                return (
+                  <span
+                    className={`text-sm text-white font-semibold px-2 py-1 rounded-md ${
+                      data.row?.original.statusId === 1
+                        ? 'bg-green-500'
+                        : 'bg-red-500'
```

Comment:
Can we use [Badge](https://ds.mochiui.com/?path=/docs/data-display-badge--docs) component here?